pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -169,6 +164,11 @@ func (c *VPCConfig) getAllowedConnsPerDirection(isIngress bool, capturedNode Nod
 			}
 			allLayersRes[peerNode] = routerConnRes
 		}
+		loadBalancerRule := c.getLoadBalancerRule(src, dst)
+		if !isIngress && loadBalancerRule != nil && loadBalancerRule.Deny() {
+			allLayersRes[peerNode] = NoConns()
```

Comment:
should there be another layer for `LB`? 
here you assign the result to `allLayersRes`, but it is not reflected in any of the `perLayerRes`.

Reply:
opened a separate issue #793